source/_integrations/braviatv.markdown

```diff
@@ -39,7 +39,9 @@ The integration supports `remote` platform. The remote allows you to send key co
 The commands that can be sent to the TV depends on the model of your TV. To display a list of supported commands for your TV, call the service `remote.send_command` with non-valid command (e.g. `Test`). A list of available commands will be displayed in [Home Assistant System Logs](https://my.home-assistant.io/redirect/logs).
 
 {% details "Some commonly used commands" %}
+
 Up, Down, Left, Right, Confirm, Return, Home, Exit, Rewind, Forward, ActionMenu, SyncMenu, Num0, Num1, Num2, Num3, Num4, Num5, Num6, Num7, Num8, Num9
```

Comment:
Please restore the previous list. A alphabetically list is easier to scan than single line with tons of options.